### DISCUSSION TITLE
right sidebar to show python code indexed by auto-functions

### DISCUSSION THREAD

**Discussion by mluerig** [pradyunsg/furo#326] [Ideas] [answered]
Would it make sense to list all functions in a specific document in the right sidebar that were parsed by the "auto" (module/class/...) functions of sphinx for easier navigation?

**Comment by pradyunsg** [comment, marked as answer]
Yes, it definitely would! See https://github.com/sphinx-doc/sphinx/issues/6316, which is the upstream tracking issue, since the auto-discovery logic needs to inject the items into the table of contents that Furo is merely rendering in a nice way. :)

In other words, yes, and the change needs to happen on Sphinx's end.
